float64 yaw_command
bool avoidance_flag
